cid/helpers/athena.py

```diff
@@ -50,10 +50,12 @@ def CatalogName(self) -> str:
                 default_catalog = [d for d in glue_data_catalogs if d['CatalogName'] == self.defaults.get('CatalogName')]
                 if not len(default_catalog):
                     # Ask user
-                    self._CatalogName = questionary.select(
-                        "Select AWS DataCatalog to use",
+
+                    self._CatalogName = get_parameter(
+                        param_name = 'glue_data_catalog',
```

Comment:
Same as above for using dashes instead of underscores

Reply:
yes